demos/python_demos/bert_question_answering_embedding_demo/bert_question_answering_embedding_demo.py

```diff
@@ -0,0 +1,338 @@
+#!/usr/bin/env python3
+
+"""
+ Copyright (c) 2020 Intel Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+import sys
+import os
+import time
+import logging as log
+from argparse import ArgumentParser, SUPPRESS
+
+import numpy as np
+
+from openvino.inference_engine import IENetwork, IECore
+
+sys.path.append(os.path.join(os.path.dirname(os.path.dirname(os.path.abspath(__file__))), 'common'))
+from tokens_bert import text_to_tokens, load_vocab_file
+from html_reader import get_paragraphs
+
+def build_argparser():
+    parser = ArgumentParser(add_help=False)
+    args = parser.add_argument_group('Options')
+    args.add_argument('-h', '--help', action='help', default=SUPPRESS, help='Show this help message and exit.')
+    args.add_argument("-i", "--input", help="Required. Urls to a wiki pages with context",
+                      required=True, type=str)
+    args.add_argument("--par_num", help="Optional. Number of contexts filtered in by embedding vectors",
```

Comment:
What does par in `par_num` stand for?

And I would paraphrase the help message: `Optional. Number of best (closest) contexts selected`

Reply:
help message changed.

Reply:
That's still unclear what par in `par_num` stands for. May be name the flag `-n` or `--best_n`

Reply:
ok